tests/nosetests/pyanaconda_tests/simple_ui_test.py

```diff
@@ -156,3 +197,30 @@ def gui_test(self, window_group, window, meh, builder, proxy_getter):
                 'UserSpoke'
             ]}
         )
+
+    @patch_dbus_get_proxy
+    @patch("pyanaconda.ui.gui.Gtk.Builder")
+    @patch("pyanaconda.ui.gui.meh")
+    @patch("pyanaconda.ui.gui.MainWindow")
+    @patch("pyanaconda.ui.gui.ANACONDA_WINDOW_GROUP")
+    def gui_random_order_test(self, window_group, window, meh, builder, proxy_getter):
+        """Test if GUI StandaloneSpokes don't have the same priority.
+
+        If we have the same priority for two standalone spokes which anchor to the same hub
+        then we are depending on sorted function implementation which can differ between
+        architectures and python versions.
+
+        This already happened in the past:
+        https://bugzilla.redhat.com/show_bug.cgi?id=929177
+        """
+        # Create the interface.
+        from pyanaconda.ui.gui import GraphicalUserInterface
+        self.interface = GraphicalUserInterface(self.storage, self.payload)
```

Comment:
Please, extend the existing tests. Don't copy-paste them.

Reply:
I'm seeing this as another test. The first test is testing if the spokes are correctly assigned to hubs and they are there. This new test is testing if the spokes are explicitly positioned.
It feels like overloading test cases.